web/src/beta/lib/reearth-ui/components/Icon/icons.ts

```diff
@@ -14,6 +14,7 @@ import CaretDoubleLeft from "./Icons/CaretDoubleLeft.svg?react";
 import CaretDoubleRight from "./Icons/CaretDoubleRight.svg?react";
 import CaretDown from "./Icons/CaretDown.svg?react";
 import CaretLeft from "./Icons/CaretLeft.svg?react";
+import CaretRight from "./Icons/CaretRight.svg?react";
```

Comment:
Please update this icon using the latest one in figma, it's graphic was not been located in the center.